frontend/src/components/login/login-input/LoginInput.tsx

```diff
@@ -77,6 +95,19 @@ const Login = () => {
     }
   }
 
+  async function sgidLogin() {
+    setErrorMsg('')
+    try {
+      const authUrl = await getSgidUrl()
+      if (authUrl) {
+        window.location.assign(authUrl)
```

Comment:
Do we use `assign` because we need `authUrl` to be in our history? Or is this suggested by the integration guide / other PRs?

Reply:
I took reference for this from the [FormSG sgid integration](https://github.com/opengovsg/FormSG/blob/856d01ddb8a5953fd68077125540674d4efd2138/frontend/src/features/login/SgidLoginPage.tsx#L45). The way I rationalised it was that if a user accidentally clicks on the Singpass login option, they should be able to navigate back to login page that they came from.

Reply:
Cool, thank you for sharing what you had in mind and also the reference!